abci/example/kvstore/kvstore.go

```diff
@@ -45,6 +45,10 @@ type Application struct {
 	// validator set
 	valUpdates         []types.ValidatorUpdate
 	valAddrToPubKeyMap map[string]cryptoproto.PublicKey
+
```

Comment:
@jmalicevic please review the detail of this file. All changes here were conflicts I have to solve manually. Please make sure I didn't make any mistake.

Reply:
Reviewed and updated.. All good!